mixer/adapter/stackdriver/contextgraph/contextgraph.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package contextgraph adapter for Stackdriver Context API.
+package contextgraph
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	contextgraph "cloud.google.com/go/contextgraph/apiv1alpha1"
+	gax "github.com/googleapis/gax-go"
+
+	"google.golang.org/api/option"
+	contextgraphpb "google.golang.org/genproto/googleapis/cloud/contextgraph/v1alpha1"
```

Comment:
style nit: we should separate out the istio imports section from the third-party. these two lines belong with the others.

Reply:
Will do.

Reply:
Done